.lintstagedrc.js

```diff
@@ -11,27 +11,32 @@ module.exports = {
   // as recommended by lint-staged.
   //
   // https://github.com/okonet/lint-staged#how-can-i-ignore-files-from-eslintignore
-  '*.{cjs,js,mjs}': async (files) => {
-    const filesToLint = await removeESLintIgnoredFiles(files)
-    return [`npm run lint:js:cli -- --fix ${filesToLint}`]
-  },
-  '*.scss': ['npm run lint:scss:cli -- --fix']
+  '*.{cjs,js,mjs}': filterTask('npm run lint:js:cli -- --fix'),
+  '*.scss': 'npm run lint:scss:cli -- --fix'
```

Comment:
Shouldn't this set the `allow-empty-input` flag that would fix the error the PR was set to solve? Happy to have that flag as part ot `lint:scss:cli` so we don't have to think about it going forward :) 

```suggestion
  '*.scss': 'npm run lint:scss:cli -- --fix --allow-empty-input'
```

Reply:
Well spotted, I took this out when I ran it through `filterTask()`

But that's gone now